cluster/node-pools/master-default/userdata.yaml

```diff
@@ -865,3 +867,26 @@ write_files:
       {{- if eq .Cluster.ConfigItems.enable_encryption "true" }}
           - identity: {}
       {{- end }}
+
+  # Enabling EventRateLimit admission plugin
+  # Reference: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#eventratelimit
+  - owner: root:root
+    path: /etc/kubernetes/config/admission-config.yaml
+    permission: '0400'
+    content: |
+      apiVersion: apiserver.config.k8s.io/v1
+      kind: AdmissionConfiguration
+      plugins:
+        - name: EventRateLimit
+          path: /etc/kubernetes/config/event-config.yaml
+
+  - owner: root:root
+    path: /etc/kubernetes/config/event-config.yaml
+    permission: '0400'
+    content: |
+      apiVersion: eventratelimit.admission.k8s.io/v1alpha1
+      kind: Configuration
+      limits:
+        - type: Server
+          qps: 500
+          burst: 1000
```

Comment:
From the Grafana dashboard, the api server reached a peak req/s of 1.81k before crashing. I've used that as a reference here to define these limits, but they are debatable, of course.

Reply:
You could also make the `qps` and `burst` values configurable. This way we can easily change them when we notice that there are better values.

Reply:
Good idea, I'll add the `config-items`!

Reply:
Added!